src/constants.js

```diff
@@ -19,3 +25,5 @@ export const DAYS_OF_WEEK = [
   "Friday",
   "Saturday",
 ];
+export const JOB_TYPE = ["Part-time", "Full-time", "Shift based"];
+export const SALARY_TYPE = ["Hourly", "Yearly"];
```

Comment:
I see. @ericfu666  and you will both work on this field. 
To avoid conflict, I will create a PR by my side to define the constants here. Could you take a look at the new PR, and apply the constants then?